src/Model/Cart.java

```diff
@@ -0,0 +1,81 @@
+package Model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import API.CartService;
+import Model.MailingList.EmailSender;
+import Model.paymentMethods.FinancialPayment;
+
+
+/**
+ * @author Jordan
+ * shopping cart
+ */
+public class Cart implements CartService{
+
+	protected List<Item> products;
+	private CustomerService userAccount;
+	
+	public Cart(CustomerService userAccount){
+		products=new ArrayList<Item>();
+		this.userAccount=userAccount;
+	}
+	
+	public void add(Product prod, int quantity){
+		products.add(new Item(quantity, prod));
```

Comment:
But, this is the user's cart. Each Customer has in her account a reference to her cart (the cart also have a reference to his owner, userAccount). notice that for add a product you need before know the user. Example:
newCustomer1.getCart().add(web.searchProduct("Oferta del Bootcamp"), 1);